geopandas/_block.py

```diff
@@ -151,10 +151,15 @@ def _astype(self, dtype, copy=False, errors='raise', values=None,
         raise on an except if raise == True
         """
 
-        if dtype != np.object_:
+        if dtype == np.object_:
+            values = self.values
+        elif dtype == str:
+            values = np.array(list(map(str, self.values)))
```

Comment:
We *could* implement a vectorized 'to_wkt' function (which might also be useful later for certain export that uses WKT, although not directly sure that will be the case)